artifactory.py

```diff
@@ -640,7 +640,17 @@ def __iter__(self):
         return self.iterator
 
 
-class _ArtifactoryAccessor(pathlib._Accessor):
+_artifactory_access_parent_class = None
+try:
+    _artifactory_access_parent_class = pathlib._Accessor
+except AttributeError:
+    _artifactory_access_parent_class = (
+        pathlib.WindowsPath if os.name == "nt" else pathlib.PosixPath
+    )
```

Comment:
I think this will work for 3.7 as well, or? 
```suggestion
_artifactory_access_parent_class = (
        pathlib.WindowsPath if os.name == "nt" else pathlib.PosixPath
    )
```

Reply:
Yes, you are right.